### DISCUSSION TITLE
https://api.vercel.com/v5/now/deployments/dpl_DxStUh57iKkh5cwy4JHJpz8y6Es6/edge it is a url to create Edge Config per project not working

### DISCUSSION THREAD

**Discussion by sahildhanawadeofficial** [vercel/community#2104] [Help] [answered]
https://api.vercel.com/v5/now/deployments/dpl_DxStUh57iKkh5cwy4JHJpz8y6Es6/edge it is a url to create Edge Config per project not working its saying This Request has triggered an infinite loop. statusCode 508

**Comment by amyegan** [comment, marked as answer]
Hi @sahildhanawadeofficial! It looks like you might need to adjust the URL. You can find more info in the documentation about [Managing Edge Configs with Vercel REST API](https://vercel.com/docs/concepts/edge-network/edge-config/vercel-api#)
